lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -2186,6 +2186,35 @@ static bool isValueChangingCast(TypeRef srcTy, TypeRef destTy) {
   return false;
 }
 
+/// Optimize away redundant ClipNodes.
+/// - First, we  basically turn "Clip(Clip(Clip(A)))" to "Clip(A)".
+/// - Then we remove the Clip at the output of transform/indexing nodes as they
```

Comment:
> Then we remove the Clip at the output of transform/indexing nodes as the

Is the logic that's described here implemented somewhere else? I don't see it in this function. If it's not part of this function we should remove this comment